src/mirco_evaluate.cpp

```diff
@@ -21,10 +21,11 @@
 #include "mirco_nonlinearsolver.h"
 
 
-void MIRCO::Evaluate(double& pressure, double Delta, double LateralLength, double GridSize,
```

Comment:
This will break the MIRCO integration in BACI. @tsarikahin please have a look together with @RShaw026 to assess the effect of this changes also on the BACI integration. If you make a change that breaks the API for BACI, then you need to also bring those changes to BACI.

Reply:
Yes, I already talked to Rishav about that and he mentioned all those challenges. But I believe it is a  necessary step for the future since in `mirco_evaluate`, no information is returned to the main file.